### PR TITLE
feat(metrics): add additional labels to metrics

### PR DESCRIPTION
**Pull request by halleystar**
support #259

### CODE REVIEW

**Comment by halleystar** [comment]
I resubmitted a PR, please code review for me

File: pkg/exporter/cmd/config.go [hunk 1]
```diff
@@ -16,7 +16,8 @@ type InspServerConfig struct {
 }
 
 type MetricsConfig struct {
-	Probes []ProbeConfig `yaml:"probes" mapstructure:"probes" json:"probes"`
+	Probes           []ProbeConfig `yaml:"probes" mapstructure:"probes" json:"probes"`
+	AdditionalLabels string        `yaml:"additionalLabels" mapstructure:"additionalLabels" json:"additionalLabels"`
```
Comment (BSWANG) [inline pkg/exporter/cmd/config.go:20]: AdditionalLabels  should be a string list?

```
metrics:
  additionalLabels:
  - name1=value1
  - name2=${labels:app}
  - name3=prefix_${labels:owner}_suffix
```

File: pkg/exporter/probe/legacy.go [hunk 2]
```diff
@@ -12,9 +13,63 @@ import (
...
+		return []string{}
+	}
+
+	var values []string
+	podLabels := entity.GetLabels()
+	for _, labelValue := range AdditionalLabelValueExpr {
+		if strings.Contains(labelValue, "${") && strings.Contains(labelValue, "}") {
+
+			//get the first ${ and last } string in labelValue
+			s := labelValue[strings.Index(labelValue, "${")+2 : strings.LastIndex(labelValue, "}")]
```
Comment (BSWANG) [inline pkg/exporter/probe/legacy.go:51]: use regex to get the experssion
```
        matches := regexp.MustCompile(`\${(.*?)}`).FindAllStringSubmatch(text, -1)
```

**Review by BSWANG — CHANGES_REQUESTED** [review]

File: pkg/exporter/cmd/config.go (hunk 1 above)
Comment (halleystar) [reply pkg/exporter/cmd/config.go:20]: According to the format in the issue specification, it is indeed a string. Of course, adding '-' in front is an array. Do I to modify it to add '-' in front

File: pkg/exporter/probe/legacy.go (hunk 2 above)
Comment (halleystar) [reply pkg/exporter/probe/legacy.go:51]: ok

File: pkg/exporter/probe/legacy.go [hunk 3]
```diff
@@ -12,9 +14,64 @@ import (
...
+	for additionalKV := range additionalLabels {
+		labelKVPair := strings.Split(additionalLabels[additionalKV], "=")
+		StandardMetricsLabels = append(StandardMetricsLabels, strings.TrimSpace(labelKVPair[0]))
+		AdditionalLabelValueExpr = append(AdditionalLabelValueExpr, strings.TrimSpace(labelKVPair[1]))
+	}
+
+	return nil
+}
+
+func BuildAdditionalLabelsValues(entity *nettop.Entity) []string {
```
Comment (BSWANG) [inline pkg/exporter/probe/legacy.go:39]: Please add an unittest for this function

**Review by BSWANG — CHANGES_REQUESTED** [review]

File: pkg/exporter/probe/legacy.go (hunk 3 above)
Comment (halleystar) [reply pkg/exporter/probe/legacy.go]: ok, I completed a unit test

**Review by BSWANG — APPROVED** [review]

**Comment by BSWANG** [comment]
LGTM, Please fix the lint issue.

**Comment by halleystar** [comment]
sorry for that. i have already fixed that.

**Pull request merged**